x-pack/plugins/code/server/indexer/lsp_indexer.ts

```diff
@@ -231,8 +234,13 @@ export class LspIndexer extends AbstractIndexer {
         this.log.debug(`Unsupported language. Skip symbols and references indexing.`);
       }
     } catch (error) {
-      this.log.error(`Index symbols or references error. Skip to file indexing.`);
-      this.log.error(error);
+      if (error instanceof ResponseError && error.code === LanguageServerNotInstalled) {
```

Comment:
I was thinking maybe handling this not installed exception in the lsp controller here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/code/server/lsp/controller.ts#L96, while leaving the indexer to be transparent to any LSP related data except their APIs.

But I guess there is not too much difference here, we can do something later.